travis-scripts/ffosxbuild.sh

```diff
@@ -96,10 +90,7 @@ sed -i -e "s/CFBundleGetInfoString.*/CFBundleGetInfoString = \"$CFBundleGetInfoS
 # Replace version strings in Info.plist
 sed -i -e "s/CFBundleShortVersionStringChangeMe/$CFBundleShortVersionString/g" $outdir/Contents/Info.plist
 sed -i -e "s/CFBundleGetInfoStringChangeMe/$CFBundleGetInfoString/g" $outdir/Contents/Info.plist
-sed -i -e "s/CFBundleVersionChangeMe/$FONTFORGE_VERSIONDATE_RAW/g" $outdir/Contents/Info.plist
-
-# Remove the temp file
-rm /tmp/fontforge-config-sorted
+sed -i -e "s/CFBundleVersionChangeMe/$FONTFORGE_VERSION/g" $outdir/Contents/Info.plist
```

Comment:
Since `VERSION_MAJOR` and `VERSION_MINOR` aren't going away it seems as if *VMa.VMi.V* would be [more compliant](https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleversion) here. But I guess if we've been doing it wrong so far there's an argument for consistency.

Reply:
VERSION_MAJOR/MINOR are mostly on the way out too, I just didn't bother to remove it from here, because they're also used for the soversions.

As for why, those versions are/were inconsistently bumped, plus we've been using a date as the version for years now

Reply:
We are going to need to fix that someday. I am not happily anticipating our adventures with epochs.

Reply:
Yeah, that's fair. I'll leave fixing that out of this PR though.